src/Draco.Compiler/Internal/Binding/Binder_Expression.cs

```diff
@@ -4,6 +4,7 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Linq;
 using Draco.Compiler.Api.Diagnostics;
+using Draco.Compiler.Api.Semantics;
```

Comment:
If you don't bring in this namespace, you won't need all those aliases, there's a reason I kept this namespace outta here

Reply:
There is no aliases ?